pkg/generic/thrift/json.go

```diff
@@ -260,3 +271,26 @@ func removePrefixAndSuffix(buf []byte) []byte {
 	}
 	return buf
 }
+
+func jsonWriter(ctx context.Context, body *gjson.Result, typeDsc *descriptor.TypeDescriptor, opt *writerOption, bw *thrift.BufferWriter) error {
+	val, writer, err := nextJSONWriter(body, typeDsc, opt)
+	if err != nil {
+		return fmt.Errorf("nextWriter of field[%s] error %w", typeDsc.Name, err)
+	}
+	if err = writer(ctx, val, bw, typeDsc, opt); err != nil {
+		return fmt.Errorf("writer of field[%s] error %w", typeDsc.Name, err)
+	}
+	return nil
+}
+
+func structReader(ctx context.Context, typeDesc *descriptor.TypeDescriptor, opt *readerOption, br *thrift.BufferReader) (v interface{}, err error) {
+	resp, err := readStruct(ctx, br, typeDesc, opt)
+	if err != nil {
+		return nil, err
+	}
+	respNode, err := sonic.Marshal(resp)
```

Comment:
sonic‘s default config is different from jsoniter. You should set `Config.EscapeHTML=true`, to be compatible with non-streaming mode?

Reply:
I'll also replace jsoniter used by non-streaming mode by sonic with  `Config.EscapeHTML=true`